scripts/apps/archive/directives/ArchivedItemKill.js

```diff
@@ -27,12 +27,12 @@ export function ArchivedItemKill(authoring, api, notify, gettext) {
                         notify.error(gettext('Failed to apply kill template to the item.'));
                     });
                 }, function (response) {
-                    if (response.data._message) {
-                        notify.error(response.data._message);
-                    } else {
-                        notify.error(gettext('Unknown Error: Cannot kill the item'));
-                    }
+                if (response.data._message) {
```

Comment:
Yeah, if I "fix" this, ESLint throws an error. Probably needs some reformatting. It's an odd one.